### ISSUE TITLE
mqtt message not as expected

### ISSUE THREAD

**Issue by grannysmith59** [Ken98045/On-Guard#6]
When setting motion payload as {Confidence} the payload received was 85} rather than just 85

**Comment by Ken98045** [comment]
Yes, I fixed that and put up a revised release this morning.  I noticed it last night, but it was just too late at night to put up another release.  Apparently I screwed up the release anyway since it was missing the setup directory.
